templates/components/text-column/text-column.scss

```diff
@@ -419,3 +419,50 @@
 	background: var(--color-black);
 	outline: 4mm solid var(--color-black);
 }
+
+.theme--lest-siden-sist {
+	.alf-block, p, ul, ol, li {
+		line-height: $base-line-height;
+		font-size: 7.9pt;
+		margin-top: $base-line-height;
+		font-family: SharpGroteskMedium20;
+
+		b {
+			font-size: 8.9pt;
+			font-family: SharpGroteskMedium20;
+		}
+
+		.row {
+			text-indent: 0;
+		}
+	}
+
+	.alf-block:first-child p {
+		margin-top: 0;
+	}
+
+	.alf-block:nth-child(even) {
+		margin-left: 0.9mm;
+	}
+
+	.alf-block:nth-child(odd)::before {
+		font-family: SharpGroteskMedium20;
+		content: '\25ba';
+		display: block;
+		position: absolute;
+		left: -4.85mm;
+		color: var(--color-base-dark);
+		@media print {
+			color: var(--color-base-dark-cmyk);
```

Comment:
Could you instead of doing this, add the `--color-base-dark` to the `@media print` definition in `themes/dn/_styles.scss`?

Reply:
Changed that now

Reply:
But looking a bit more at the styling in `_styles.scss` I'm not sure this will work, since we in the various theme sections overwrite both `--color-base-dark` and `--color-base-dark-cmyk` with new values taken from the SCSS variables.

Should probably rewrite all of these components to not use `...-cmyk` colors anywhere, and only use those in a `@media print` block at root level. But I'm not sure this PR is the right place for that.

Reply:
Yes, the theme stuff should be updated too.

Reply:
I'm not sure how the other theme stuff is used, and I don't have examples of those templates locally to test the changes. Is it okay if we save that for another PR?